style.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Poppins:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&family=Poppins:ital,wght@0,200;0,700;1,500&family=Raleway:wght@200;400;600&display=swap');
```

Comment:
- [ ] [OPTIONAL] I'd heavily like to apologize 🙏 as it seems like I haven't included all the proper font-weights in my previous comment and it's causing some texts to be very light/thin. But I've checked it now so kindly consider replacing it with the one below:
- `@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;1,700&display=swap');` . This has all the weights ranging from 300-600 as specified in the design.